ukvm/ukvm_cpu_aarch64.c

```diff
@@ -105,3 +105,15 @@ void aarch64_setup_memory_mapping(uint8_t *va_addr, uint64_t ram_size,
         phy_space_size -= map_size;
     }
 }
+
+void aarch64_mem_size(size_t *mem_size) {
+    size_t mem;
+    mem = (*mem_size / PMD_SIZE) * PMD_SIZE;
```

Comment:
Sorry, here seems use round_down to calculate memory size. From the conversation history I have found a concern about round_up/round_down. IMO, if use round_down, if the application need at less 11MB memory to run, but we'll allocate only 10MB for it. I think the application could not work properly.

Reply:
I don't think it matters too much, since this is the _maximum_ memory size usable by the guest, so I'd not expect people to try and trim it down to _exact_ figures. In any case, it'll only ever get rounded down by 1MB and you will get a warning. We can always revisit this later.

Reply:
Ok, it's not a big problem.

Reply:
@Weichen81 I don't know AARCH64 details, this code is very similar to its `x86_64` counterpart, and allocates only in 2MB chunks.  If it is possible on AARCH64 to allocate smaller amounts (such as 512KB or 1MB), we should adjust this code to allow that.  if a unikernel needs 11MB, the only way atm is to allocate 12MB for it, due to the 2MB chunks.

Reply:
@hannesm 2MB is the minimal granularity for memory allocation on AARCH64. Allocate 512KB or 1MB granularity  may create empty "hole" in mapping table. So you don't need to adjust  your current code for AARCH64.